provider/machinepool/classic/machine_pool_resource.go

```diff
@@ -210,6 +210,12 @@ func (r *MachinePoolResource) Schema(ctx context.Context, req resource.SchemaReq
 				ElementType: types.StringType,
 				Optional:    true,
 			},
+			"aws_tags": schema.MapAttribute{
+				Description: "Keys and values that the machine pool provisioner will add as AWS tags to all AWS " +
```

Comment:
I see you've taken the description from rosa cli, would you mind to update it to be consistent with other `tags` attributes from terraform resources?
https://github.com/terraform-redhat/terraform-provider-rhcs/blob/main/provider/machinepool/hcp/aws_node_pool.go#L39